File: easy/jungle-camping/JungleCamping.java
import java.util.Map;
import java.util.Scanner;

public class JungleCamping {
    public static void main(String[] args) {
        final var animals = Map.of(
            "Grr", "Lion ", "Rawr", "Tiger ", "Ssss", "Snake ", "Chirp", "Bird ");
          
        new Scanner(System.in).tokens().map(animals::get).forEach(System.out::print);
    }
}

File: easy/jungle-camping/JungleCamping.py
"""
animals = {"Grr": "Lion", "Rawr": "Tiger", "Ssss": "Snake", "Chirp": "Bird"}

noise = input().split()
print(' '.join(map(animals.get, noise)))

"""


print(input().replace("Grr","Lion").replace("Rawr","Tiger").replace("Chirp","Bird").replace("Ssss","Snake"))
